nixos/doc/manual/installation/installing.xml

```diff
@@ -29,13 +29,14 @@
   </para>
 
   <para>
-   You are logged-in automatically as <literal>root</literal>. (The
-   <literal>root</literal> user account has an empty password.)
+   You are logged-in automatically as <literal>live</literal>.
+   The <literal>live</literal> user account has an empty password so you
+   can use <command>sudo</command> without a password.
   </para>
 
   <para>
    If you downloaded the graphical ISO image, you can run <command>systemctl
-   start display-manager</command> to start KDE. If you want to continue on the
+   start display-manager</command> to start the DE. If you want to continue on the
```

Comment:
I think we should write this out instead of using a relatively rare abbreviation — maybe "the graphical environment".